wire-grpc-tests/src/test/java/com/squareup/wire/MockRouteGuideService.kt

```diff
@@ -55,6 +56,44 @@ class MockRouteGuideService : RouteGuideGrpc.RouteGuideImplBase(), TestRule {
     script.add(action)
   }
 
+  fun enqueueReceiveNote(message: String) {
+    enqueue(Action.ReceiveMessage(RouteNote.newBuilder()
```

Comment:
Doesn't `RouteNote(message = message)` work? You shouldn't need to use `newBuilder()`.

Reply:
This is a Google protobuf!